### ISSUE TITLE
Set number format to "text"

### ISSUE THREAD

**Issue by sinhavartika** [SheetJS/js-xlsx#726]
Text number format is represented with string '@'.
**Here is the code I have been trying** (using [this](https://gist.github.com/SheetJSDev/a38ce44ca4e4a1012f3b) as an example) 
function _jsonToXlsx(data, options){
	var defaults = {
		start: {
			cell : 0,
			row : 0
		}
	};
	var options = options || defaults;

	var ws_name = "sheetname";

	/* set up workbook objects -- some of these will not be required in the future */
	var wb = {}
	wb.Sheets = {};
	wb.Props = {};
	wb.SSF = {};
	wb.SheetNames = [];

	/* create worksheet: */
	var ws = {};

	/* the range object is used to keep track of the range of the sheet */
	var range = {s: {c:options.start.cell, r:options.start.row}, e: {c:0, r:0 }};

	/* Iterate through each element in the structure */
	for(var R = 0; R != data.length; ++R) {
	  if(range.e.r < R) range.e.r = R;
	  for(var C = 0; C != data[R].length; ++C) {
	    if(range.e.c < C) range.e.c = C;

	    /* create cell object: .v is the actual data */
	    var cell = { v: data[R][C] };
	    if(cell.v == null) continue;

	    /* create the correct cell reference */
	    var cell_ref = XLSX.utils.encode_cell({c:C,r:R});

	    /* determine the cell type */
	    cell.t = 's';

	    /* set the format */
	    cell.z = '@';
	    
	    /* add to structure */
	    ws[cell_ref] = cell;
	    
	    /* this call updates the formatted text field (`.w`) */
	    XLSX.utils.format_cell(ws[cell_ref]);
	    
	    console.log(ws[cell_ref]);
	  }
	}
	ws['!ref'] = XLSX.utils.encode_range(range);

	/* add worksheet to workbook */
	wb.SheetNames.push(ws_name);
	wb.Sheets[ws_name] = ws;
	return wb;
}

**The console log outputs**  : { v: '00:00:00.02', t: 's', z: '@', w: '00:00:00.02' }

But the sheet still shows the cell to have _General_ number format instead of _Text_.

I am using version 0.8.7 at node.
You can find the sheet in the attachment. 
[audio_segments_template.xlsx](https://github.com/SheetJS/js-xlsx/files/1138043/audio_segments_template.xlsx)

**Comment by reviewher** [comment]
Can you update (current version 0.10.8) and check if it's still an issue in the latest version?

---

I believe this was fixed sometime between 0.8.7 and today's 0.10.8 release.  To verify, if you have access to node, install the module globally run the included `xlsx` script:

```bash
$ npm i -g xlsx
$ xlsx --dev -X audio_segments_template.xlsx
```

This reads the original file and generates a new file `audio_segments_template.xlsx.xlsx`.  The generated file appears to preserve the text format

**Comment by sinhavartika** [comment]
Thanks this solved the issue.
